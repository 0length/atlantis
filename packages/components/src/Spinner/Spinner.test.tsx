import React from "react";
import renderer from "react-test-renderer";
import { Spinner } from ".";

it("renders the spinner", () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the small spinner", () => {
  const tree = renderer.create(<Spinner small={true} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders the inline spinner", () => {
  const tree = renderer.create(<Spinner inline={true} />).toJSON();
  expect(tree).toMatchSnapshot();
});
