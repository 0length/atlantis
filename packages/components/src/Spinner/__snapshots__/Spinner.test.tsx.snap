// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the inline spinner 1`] = `
<div
  className="spinner inline"
/>
`;

exports[`renders the small spinner 1`] = `
<div
  className="spinner"
/>
`;

exports[`renders the spinner 1`] = `
<div
  className="spinner"
/>
`;
